Silder add

ahmrafi22 opened the pull request:
Description:

1. Added the Slider component in /components/ui/.
2. Installed @radix-ui/react-slider using pnpm add @radix-ui/react-slider.
3. Added a Slider preview in preview/components/.
4. Registered the preview in config/components.ts:
   i.slider-style-default for the default variant.
   ii.slider-style-variants for additional examples.
5. Note: Installation instructions were not included in the documentation

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Introduced a customizable slider component with multiple styles that enhance interactivity and accessibility.
  - Showcased interactive examples for volume and brightness control.

- **Documentation**
  - Added detailed documentation including installation instructions, usage examples, and API references.

- **Chores**
  - Updated configuration and export settings for streamlined integration.
  - Integrated a new dependency to support advanced slider functionality.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

ariflogs commented on content/docs/components/slider.mdx:
```diff
@@ -0,0 +1,68 @@
+
+## Customization
+
+### Value Display
```
we don't need to write separate customization guide as it should work as as radix UI, so just include API reference at the top. 
Example: https://ui.shadcn.com/docs/components/slider

ariflogs commented on preview/components/slider-style-default.tsx:
```diff
@@ -0,0 +1,23 @@
+  return (
+    <div className="p-6 max-w-md mx-auto">
+      <div className="mb-6">
+        <label 
```
use our `Label` component

ariflogs reviewed (changes requested):
Hi @ahmrafi22, I have left some comments on your code. please check them out.
thanks.

ahmrafi22 commented on components/ui/Slider/Slider.tsx:
Removed showing the value inside the component, so all custom logic is removed , removed the grid structure and kept only two variants: thin and default. The logic for two thumbs (range slider) implement properly like shadCN

ahmrafi22 commented:
Removed showing the value inside the component, so all custom logic is removed , removed the grid structure and kept only two variants: thin and default. The logic for two thumbs (range slider) implement properly like shadCN

Used Label from component and added proper installation guide
